executor/foreign_key.go

```diff
@@ -815,3 +865,87 @@ func genWhereConditionAstForMultiColumn(cols []*model.ColumnInfo, fkValues [][]t
 		List: valueList,
 	}
 }
+
+// String implements the RuntimeStats interface.
+func (s *FKCheckRuntimeStats) String() string {
+	buf := bytes.NewBuffer(make([]byte, 0, 32))
+	buf.WriteString("total:")
+	buf.WriteString(execdetails.FormatDuration(s.Total))
+	if s.Check > 0 {
+		buf.WriteString(", check:")
+		buf.WriteString(execdetails.FormatDuration(s.Check))
+	}
+	if s.Lock > 0 {
+		buf.WriteString(", lock:")
+		buf.WriteString(execdetails.FormatDuration(s.Lock))
+	}
+	if s.Keys > 0 {
+		buf.WriteString(", foreign_keys:")
+		buf.WriteString(strconv.Itoa(s.Keys))
+	}
+	return buf.String()
+}
+
+// Clone implements the RuntimeStats interface.
+func (s *FKCheckRuntimeStats) Clone() execdetails.RuntimeStats {
+	newRs := &FKCheckRuntimeStats{
+		Total: s.Total,
+		Check: s.Check,
+		Lock:  s.Lock,
+		Keys:  s.Keys,
+	}
+	return newRs
+}
+
+// Merge implements the RuntimeStats interface.
+func (s *FKCheckRuntimeStats) Merge(other execdetails.RuntimeStats) {
+	tmp, ok := other.(*FKCheckRuntimeStats)
+	if !ok {
+		return
```

Comment:
any log info？

Reply:
No. The other same logic place also doesn't have a log.